pkg/machineproviders/providers/openshift/machine/v1beta1/mapping_test.go

```diff
@@ -234,20 +242,25 @@ var _ = Describe("Failure Domain Mapping", func() {
 				machines = append(machines, *machine)
 			}
 
-			mapping, err := mapMachineIndexesToFailureDomains(logger.Logger(), machines, in.replicas, failureDomains)
+			mapping, err := mapMachineIndexesToFailureDomains(logger.Logger(), machines, in.replicas, failureDomains, templateFailureDomain)
```

Comment:
Should this be coming from the test case?
```suggestion
			mapping, err := mapMachineIndexesToFailureDomains(logger.Logger(), machines, in.replicas, failureDomains, tc.templateFailureDomain)
```